src/commissaire_service/investigator/__init__.py

```diff
@@ -170,7 +170,6 @@ def on_investigate(self, message, address, cluster_data={}):
                     network.name, network.type))
             transport.bootstrap(
                 address, key.path, oscmd, etcd_config, network)
-            host.status = 'inactive'
```

Comment:
Can I ask you to tweak this a bit further?
- Set `host.status` to `'disassociated'` instead of `'inactive'` here.
- In the `except` block, I think the `host.status` should be `'failed'` not `'disassociated'` since it failed bootstrapping.
- Further down where we call `container.register_node`, drop the `host.status = 'disassociated'` and rearrange the logic like so:
```
        if container_manager:
            try:
                self.request(...)
                host.status = 'active'
            except ...:
                ...
            finally:
                self.storage.save(host)
```